core-bundle/contao/templates/forms/form_wrapper.html5

```diff
@@ -6,19 +6,15 @@
     <<?= $this->hl ?>><?= $this->headline ?></<?= $this->hl ?>>
   <?php endif; ?>
 
-  <form<?php if ($this->action): ?> action="<?= $this->action ?>"<?php endif; ?> method="<?= $this->method ?>" enctype="<?= $this->enctype ?>"<?= $this->attributes ?><?= $this->novalidate ?>>
-    <div class="formbody">
-      <?php if ('get' != $this->method): ?>
-        <input type="hidden" name="FORM_SUBMIT" value="<?= $this->formSubmit ?>">
-        <input type="hidden" name="REQUEST_TOKEN" value="<?= $this->requestToken ?>">
-        <?php if ($this->maxFileSize): ?>
-          <input type="hidden" name="MAX_FILE_SIZE" value="<?= $this->maxFileSize ?>">
-        <?php endif; ?>
-      <?php endif; ?>
-      <?= $this->hidden ?>
-      <?= $this->fields ?>
-    </div>
-  </form>
+  <?php if ($this->confirmation): ?>
+    <?php $this->insert('form_confirmation', $this->arrData) ?>
+  <?php else: ?>
+    <?php $this->insert('form_inline', $this->arrData) ?>
+  <?php endif; ?>
+
+  <?php if (!$this->action): ?>
```

Comment:
I would prefer this to be a checkbox in the form settings. It can be enabled by default, so that new forms are submitted via Ajax and existing ones are not.

Reply:
I have added the checkbox in [d19de43](https://github.com/contao/contao/pull/5307/commits/d19de432f73101bd3992710a116e041e357b73e0).